_health-care/_js/_form.js

```diff
@@ -18,17 +18,60 @@ var HealthApp = (function() {
     var dateString = "";
 
     if (date.getMonth() < 10) {
-      dateString += "0"
+      dateString += "0";
     }
     dateString += (date.getMonth() + 1) + "/";
 
     if (date.getDate() < 10) {
-      dateString += "0"
+      dateString += "0";
     }
     dateString += date.getDate() + "/";
 
     return dateString + date.getFullYear();
   }
+  function extractMonetaryValueTransform(formData, arg) {
+    return Number(extractTransform(formData, arg)).toFixed(2).toString();
+  }
+  function validatePresence(input) {
+    var valid = true;
+
+    if (input.value === '') {
+      valid = false;
+    }
+
+    return valid;
+  }
+  function validateMonetaryValue(input) {
+    var valid = true;
+    var amount = input.value;
+
+    if (/[$a-zA-Z,]+/.test(amount)) {
+      valid = false;
+    }
+
+    return valid;
+  }
+  function validate(input) {
+    var valid = true;
+
+    $(input).removeClass('error');
+
+    valid = validatePresence(input);
+    if (!valid) {
+      $(input).addClass('error');
+      return false;
+    }
+
+    if ($(input).data('validation-type') === 'monetary') {
+      valid = validateMonetaryValue(input);
+    }
+
+    if (!valid) {
+      $(input).addClass('error');
+    }
+
+    return valid;
```

Comment:
Makes sense! I agree that perhaps as this expands with more cases we might want a different solution (perhaps an object with each type of validator as the key and the function as the value?).